index.html

```diff
@@ -1,15 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-    <link href="https://fonts.googleapis.com/css2?family=Rubik&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="styles/index.css" />
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Weather Report</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link href="https://fonts.googleapis.com/css2?family=Cabin+Sketch&display=swap" rel="stylesheet" />
+  <link rel="stylesheet" href="styles/index.css" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.0/css/all.min.css">
 </head>
+
 <body>
-    <script src="./src/index.js"></script>
+  <video autoplay muted loop id="bg-video">
+    <source src="assets/Timelaps.mp4" type="video/mp4" />
+    Your browser does not support the video tag.
+  </video>
+
+  <div id="page-wrapper">
+    <header>
+      <div id="header-row">
+        <h1>WEATHER REPORT</h1>
+        <img src="assets/YouAreHere.png" alt="You Are Here Icon" id="you-are-here-icon" />
+        <div id="city-name-wrapper">
+          <span id="displayed-city-name">Seattle</span>
+        </div>
+      </div>
+    </header>
+
+    <main id="bottom-layout">
+      <div class="column-left">
+        <section class="box" id="temperature-box">
+          <h2 class="temperature-title">TEMPERATURE</h2>
+          <div class="temperature__layout">
+            <button id="decreaseTempControl" class="arrow">
+              <i class="fas fa-arrow-left"></i>
+            </button>
+            <span id="tempValue">65</span>
+            <button id="increaseTempControl" class="arrow">
+              <i class="fas fa-arrow-right"></i>
+            </button>
+          </div>
+          <button id="temp-now-btn">Click Me</button>
```

Comment:
I like that you used semantic `button` tag for this element (instead of something meaningless like `span`). "Click Me" does tell me what to do, but not what clicking does. 

```suggestion
          <button id="temp-now-btn">Get City Temperature</button>
```